tests/ui/codegen/large-stack-size-issue-83060.rs

```diff
@@ -0,0 +1,19 @@
+// This test checks that allocating a stack size of 1GB or more results in a compilation failure
```

Comment:
This says "compilation failure" but it builds, according to `@ build-pass` below?

Reply:
Sorry about that!